src/extract.ts

```diff
@@ -474,8 +472,8 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
 
 function extractCustomBenchmarkResult(output: string): BenchmarkResult[] {
     try {
-        const json: CustomBenchmarkJson = JSON.parse(output);
-        return json.benchmarks.map(bench => {
+        const json: [CustomBenchmarkJson] = JSON.parse(output);
```

Comment:
This is a great call out! It actually looks like we can simply do this?

```suggestion
        const json: [BenchmarkResult] = JSON.parse(output);
```